app/models/concerns/status/safe_reblog_insert.rb

```diff
@@ -15,7 +15,9 @@ module Status::SafeReblogInsert
     #
     # The code is kept similar to ActiveRecord::Persistence code and calls it
     # directly when we are not handling a reblog.
-    def _insert_record(values, returning)
+    #
+    # https://github.com/rails/rails/blob/v7.2.1.1/activerecord/lib/active_record/persistence.rb#L238-L263
```

Comment:
I think it makes sense to keep this annotated as we update it.

Another approach I've seen is to put some sort of initializer version check in, and raise an error on any updates, which sort of forces the issue of manual verifying compatibility on every update.